stories/src/geo.stories.tsx

```diff
@@ -0,0 +1,386 @@
+import * as React from 'react'
+import {storiesOf} from '@storybook/react'
+import {Config, Plot} from '../../giraffe/src'
+
+import {PlotContainer} from './helpers'
+import {geoTable, geoTracks} from './data/geoLayer'
+import {boolean, color, number, select, withKnobs} from '@storybook/addon-knobs'
+import {ClusterAggregation} from '../../giraffe/src/types/geo'
+
+const tileServerConfiguration = {
+  tileServerUrl: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  // bingKey:
+  //   'AtqWbnKXzGMWSAsgWknAw2cgBKuGIm9XmSbaS4fSebC5U6BdDTUF3I__u5NAp_Zi',
```

Comment:
Perhaps the key should not be stored publicly.

Reply:
I have created this key solely for giraffe demo purposes. It is free and there is a quota large enough do demonstrate giraffe for potential users.
I have turned this key into a separate story book story.